tests/slow/src/test/scala/tests/feature/WorkspaceSymbolRegressionSuite.scala

```diff
@@ -7,7 +7,7 @@ import tests.BaseWorkspaceSymbolSuite
 
 class WorkspaceSymbolRegressionSuite extends BaseWorkspaceSymbolSuite {
   def workspace: AbsolutePath = Corpus.akka()
-  check("Actor", "1025 results")
+  check("Actor", "1009 results")
```

Comment:
I checked all affected tests that were changed except this one.

Actually, this change just spotted the same problem as described in #2742.